ocs_ci/helpers/dr_helpers.py

```
@@ -4532,6 +4533,14 @@ def validate_application_odf_cli(
         Skips the test on validation failure.
 
     """
+    cluster_name = config.ENV_DATA.get("cluster_name")
+    if is_hosted_cluster(cluster_name):
+        logger.info(
+            f"Skipping ODF CLI DR {action} for DRPC '{drpc_name}': "
+            f"cluster '{cluster_name}' is a hosted cluster"
```

Ruff (0.16.1)

[warning] Logging statement uses f-string

(G004)